### ISSUE TITLE
Need add More __Nonnull 

### ISSUE THREAD

**Issue by tentenlee100** [vilanovi/UIImage-Additions#14]
in 2.1.2 you add  __Nonnull in code
but you need to add more on 

```
FOUNDATION_EXTERN NSString* NSStringFromADDCornerInset(ADDCornerInset cornerInset);
+ (UIImage*)add_imageNamed:(NSString *)name tintColor:(UIColor*)color style:(ADDImageTintStyle)tintStyle;
- (UIImage*)add_tintedImageWithColor:(UIColor*)color style:(ADDImageTintStyle)tintStyle;
- (UIImage*)add_imageWithRoundedBounds;
- (UIImage*)add_imageWithCornerRadius:(CGFloat)cornerRadius;
- (UIImage*)add_imageWithCornerInset:(ADDCornerInset)cornerInset;
- (UIImage*)add_imageAddingImage:(UIImage*)image;
- (UIImage*)add_imageAddingImage:(UIImage*)image offset:(CGPoint)offset;
+ (UIImage*)add_imageWithGradient:(NSArray*)colors size:(CGSize)size direction:(ADDImageGradientDirection)direction;
+ (UIImage*)add_resizableImageWithGradient:(NSArray*)colors size:(CGSize)size direction:(ADDImageGradientDirection)direction;
- (UIImage*)add_imageWithSize:(CGSize)newSize interpolationQuality:(CGInterpolationQuality)quality;
- (UIImage *)add_imageByFixingOrientation;
```

**Comment by vilanovi** [comment]
True. Will fix on version 2.1.3

**Comment by vilanovi** [comment]
Done. Available on Cocoa Pods.
